packages/algob/src/internal/deployer.ts

```diff
@@ -101,7 +101,7 @@ class DeployerBasicMode {
     return this.algoOp.algodClient;
   }
 
-  async waitForConfirmation (txId: string): Promise<algosdk.ConfirmedTxInfo> {
+  async waitForConfirmation (txId: string): Promise<algosdk.modelsv2.PendingTransactionResponse> {
```

Comment:
Can we directly import `modelsv2` first ( `import { modelsv2 } from "algosdk"`) & directly do `modelsv2.<type>` at all places?

Reply:
yes, it is like that l. Probably missed here. Will update it.